src/main/java/data/Camera.java

```diff
@@ -163,4 +163,17 @@ public boolean equals(Object o) {
             return camera.ip == null;
         }
     }
+    
+    /**
+     * Construct string to display this camera in a modal.
+     * @return the correct string
+     */
+    public String constructModalString() {
```

Comment:
It would require making a class only for this method, which seems pretty overkill to me